BinanceExchange.API/Trade/BinTrade.cs

```diff
@@ -0,0 +1,191 @@
+            {
+                Asset = asset,
+                IsolatedSymbol = symbol,
+            });
+
+            return maxBorrowAsset.amount + freeCoins;
+        }
+
+        private static async Task<bool> _BuySellCommand(BinanceClient client, string symbol, OrderSide buySell, SideEffectType sideEffectType, string isIsolated, decimal price, decimal quant)
+        {
+            var createIsolatedOrder = await client.CreateIsolatedOrder(new CreateIsolatedOrderRequest()
+            {
+                Price = price,
+                Quantity = quant,
+                Side = buySell,
+                Symbol = symbol,
+                Type = OrderType.Limit,
+                NewOrderResponseType = NewOrderResponseType.Full,
+                TimeInForce = TimeInForce.GTC,
+                IsIsolated = isIsolated,
+                SideEffectType = sideEffectType,
+                TimeStamp = DateTime.UtcNow.Ticks
+            });
+
+            return (true);
+        }
+
+
+        /// <summary>
+        /// Send isolated account buy/sell command
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="symbol">the trading symbol (BTCUSDT...)</param>
+        /// <param name="asset">the trading asset such as BTC, USDT ...</param>
+        /// <param name="price"></param>
+        /// <returns></returns>
+        public static async Task<bool> BuyCommandIso(BinanceClient client, string symbol, decimal price, decimal quant)
+        {
+            bool res = _BuySellCommand(client, symbol, OrderSide.Buy, SideEffectType.MarginBuy, "TRUE", price, quant).Result;
+            return res;
+        }
+        public static async Task<bool> SellCommandIso(BinanceClient client, string symbol, decimal price, decimal quant)
+        {
+            bool res = _BuySellCommand(client, symbol, OrderSide.Sell, SideEffectType.AutoRepay, "TRUE", price, quant).Result;
+            return res;
+        }
+
+
+        /// <summary>
+        /// Send isolated account market buy command
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="symbol">the trading symbol (BTCUSDT...)</param>
+        /// <param name="asset">the trading asset such as BTC, USDT ...</param>
+        /// <param name="price"></param>
+        /// <returns></returns>
+        public static async Task<bool> BuyCommandMarket(BinanceClient client, string symbol, decimal quant)
+        {
+            var createIsolatedOrder = await client.CreateIsolatedOrder(new CreateIsolatedOrderRequest()
+            {
+                Quantity = quant,
+                Side = OrderSide.Buy,
+                Symbol = symbol,
+                Type = OrderType.Market,
+                NewOrderResponseType = NewOrderResponseType.Full,
+                IsIsolated = "TRUE",
+                SideEffectType = SideEffectType.MarginBuy,
+            });
+
+            return true;
```

Comment:
Same here, please inspect the  returned order

Reply:
good point - OrderId it is